src/components/Profile/components/Identification/components/SocialMediaLinks/components/LinksDialog/index.jsx

```diff
@@ -9,14 +9,18 @@ import {
 import { useState } from 'react';
 import { platforms, socialMediaInfo } from 'services/socialMedia';
 import user from 'services/user';
+import { useUserActions } from 'hooks';
 import styles from './LinksDialog.module.css';
 
 const LinksDialog = ({ links, createSnackbar, onClose, setLinks }) => {
   const [formErrors, setFormErrors] = useState([]);
   const [updatedLinks, setUpdatedLinks] = useState(links);
   const [failedUpdates, setFailedUpdates] = useState([]);
+  //const [isSubmitted, setIsSubmitted] = useState(false);
```

Comment:
Can you remove this commented out code, so that it doesn't confuse future programmers?